src/DIRAC/Resources/Computing/ARCComputingElement.py

```diff
@@ -256,6 +256,7 @@ def _writeXRSL(self, executableFile, inputs=None, outputs=None, executables=None
 (inputFiles=({executable} "{executableFile}") {xrslInputAdditions})
 (stdout="{diracStamp}.out")
 (stderr="{diracStamp}.err")
+(environment="(DIRAC_PILOT_STAMP {diracStamp}"))
```

Comment:
Janusz has been testing this and it seems this line is causing submission errors; should it be:
`(environment=("DIRAC_PILOT_STAMP {diracStamp}"))` (e.g. with the bracket outside of the string after the equals)?

Reply:
Hmm, my previous suggestion isn't quite right... The key and value should also be quoted independently:
`(environment=("DIRAC_PILOT_STAMP" "{diracStamp}"))`

Reply:
I can confirm that the submission  to ARC CE works now when setting:
`(environment=("DIRAC_PILOT_STAMP" "{diracStamp}"))`